versions

```diff
@@ -1,5 +1,5 @@
-BACKPORTS_RELEASE_TAG="RHEL_86_6043_PRERELEASE_220804.0"
-DII_KERNEL_TAG="DII_6043_prerelease"
-DII_KERNEL_HEAD="5bb330bd8e370e"
-RHEL_8.6_KERNEL_VERSION="4.18.0-372.16.1"
+BACKPORTS_RELEASE_TAG="RHEL_86_6213_PRERELEASE_220914.1"
+DII_KERNEL_TAG="PROD_ATSM_6213.0.1"
+DII_KERNEL_HEAD="e8a6ee40e1a3ff"
+RHEL_8.6_KERNEL_VERSION="4.18.0-372.26.1"
 RHEL_8.5_KERNEL_VERSION="4.18.0-348.23.1"
```

Comment:
Add similar table as ubuntu 
RHEL 8.6 building and testing 
RHEL 8.5 building but not being tested

Add Tabel for Key releases with tag information